faceIDLogin/faceIDLogin/WelcomeViewController.swift

```diff
@@ -15,6 +15,15 @@ class WelcomeViewController: UIViewController {
         // Do any additional setup after loading the view.
     }
     
+    @IBAction func btnLogout(_ sender: Any) {
+        DispatchQueue.main.async {
+            let backViewController =  self.storyboard?.instantiateViewController(withIdentifier: "ViewController") as! ViewController
```

Comment:
Acho que a melhor opção seria usar um self.dismiss(), se não me engano dessa forma você está empilhando uma nova viewController. Sua pilha deve ter Login > Home > Login novamente, o ideal é dar um dismiss na Home qdo for deslogar, assim sua pilha fica apenas com Login

Reply:
Resolvido.